src/lint.js

```diff
@@ -75,7 +75,7 @@ function runLinter (opts = {}) {
   return new Promise((resolve, reject) => {
     const cli = new CLIEngine({
       useEslintrc: true,
-      configFile: CONFIG_FILE,
```

Comment:
Need to remove wherever CONFIG_FILE came from, otherwise this will fail linting.

Reply:
done